build/web-client-rpc.md

```diff
@@ -0,0 +1,55 @@
+# Web Client X RPC
+
+If you're unsure which client to use for interacting with the Nimiq blockchain, this page offers a comparison between the Nimiq Web Client and the RPC so you can choose the tool that best fits your needs and expertise. {.subline}
+
+### TL;DR
+The Nimiq Web Client establishes direct connections with other peers in the network, making it a part of the blockchain network. In contrast, the RPC follows a client-server model, where clients connect to a single server that interacts with multiple peers on the network.
+
+<img class="object-contain max-h-[max(60vh,180px)]" src="/assets/images/protocol/network.png" alt="skip block struct" />
```

Comment:
Nice picture!

What about changing the title of the left block to "Nimiq Web Client" and the title of the right block "RPC Client".

Reply:
Soeren's idea! yes, I'm on it